com.jetbrains.Toolbox.yml

```diff
@@ -0,0 +1,65 @@
+id: com.jetbrains.Toolbox
+command: jetbrains-toolbox
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=x11
+  - --socket=ssh-auth
+  - --share=network
+  - --socket=session-bus
+  - --talk-name=org.freedesktop.secrets
```

Comment:
why is this needed?

Reply:
sorry for the late response
ssh-auth is for ssh feature in this applications
network is needed for downloading IDE
session bus and secrets is not needed anymore